### DISCUSSION TITLE
Create a plugin for Wordpress that allows me to allow my users to click on a link in a paragraph that takes them to a new Gutenberg page editor to create a new page

### DISCUSSION THREAD

**Discussion by ghost** [community/community#120292] [Programming Help]
### Select Topic Area

General

### Body

If I'm in the wrong forum, please tell me, but I need something, a plugin, a widget, something that will allow my users to click on a links in my Wordpress pages that take them to a new Gutenberg page editor which allows them to create a new page within my sites platform. Just like Wikipedia does for their users within their platform. So I host a wiki like wikipedia called OHMS Wiki, which stands for OHio Music Scene Wiki, where Ohio musicians can come and create wiki pages about the Ohio Music Scene, or about their own music careers or whatever that has to do with the music scene in Ohio. Wikipedia allows their users to click on these "red links" which takes their users to their page editor to create a new pages. How can I implement this function into my own wordpress platform. I use the default Wordpress gutenberg page and post editor, along with BetterDocs plugin. Please help, can you create a new plugin or widget that allows this function?

**Comment by ebndev** [comment]
Hi @alicia-rogers!

I'm moving your post to our Programming Help category and you may have more luck there.

Thanks for being a part of the GitHub community!
